tests/h/services/search_index/service_test.py

```diff
@@ -138,13 +134,9 @@ def test_it_does_nothing_if_the_annotation_is_deleted(
     def annotation(self, factories):
         return factories.Annotation.build()
 
-    @pytest.fixture(autouse=True)
-    def AnnotationSearchIndexPresenter(self, patch):
-        return patch("h.services.search_index.service.AnnotationSearchIndexPresenter")
```

Comment:
Same reasons as with `storage` above: this is a patch for a module-level import so it should be a module-scoped fixture